extensions/amp-story-shopping/0.1/amp-story-shopping-attachment.js

```diff
@@ -286,6 +299,12 @@ export class AmpStoryShoppingAttachment extends AMP.BaseElement {
    * @private
    */
   toggleDetailsText_(detailsContainer, shouldOpen) {
+    if (shouldOpen) {
+      this.analyticsService_.triggerEvent(
+        StoryAnalyticsEvent.SHOPPING_PRODUCT_DETAILS
+      );
```

Comment:
My bad - I wasn't clear about this. This is meant to track when a PDP (product details page) is viewed. 
I just updated the original ticket.

Reply:
Sounds good, I'll refactor the event to be triggered when the product details page is viewed.